participation/membership-tiers.md

```diff
@@ -1,2 +1,9 @@
 |Corda Network Foundation|[Document history]({{ site.github.repository_url }}/blame/master/{{page.path}})|
 
+Currently there are two membership tiers for Corda Network:
+(1) Full participant
+(2) Identity-Only / sponsored participant
+
+A **full participant** will usually sign contracts directly with R3. They will usually manage their own nodes and be part of  tightly-knit business network(s). They will receive full customer support from the Network Operator, can be a member of multiple business networks, will have voting rights and can stand for the Foundation's Board of directors.  
```

Comment:
directly with the Foundation.

Reply:
may be part of several? tightly-knit business networks.

receive full on-boarding support from the

Reply:
can we remove the future tense on this page?

Reply:
yes to all - done